src/components/anchor/anchor.scss

```
@@ -0,0 +1,30 @@
+:host {
+    display: inline;
+}
+
+.anchor {
+    display: inline;
+    margin-left: 0.25em;
+
+    color: rgb(var(--kompendium-contrast-700));
```

Stylelint (17.7.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)